samples/hosting/docker/Core_7/Receiver/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using NServiceBus;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Exceptions;
+
+class Program
+{
+    static async Task Main()
+    {
+        //required to prevent possible occurrence of .NET Core issue https://github.com/dotnet/coreclr/issues/12668
+        Thread.CurrentThread.CurrentUICulture = new CultureInfo("en-US");
+        Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
+
+        Console.CancelKeyPress += new ConsoleCancelEventHandler(OnExit);
+
+        Console.Title = "Samples.Docker.Receiver";
+
+        var endpointConfiguration = new EndpointConfiguration("Samples.Docker.Receiver");
+        #region TransportConfiguration
+        endpointConfiguration.UseTransport<RabbitMQTransport>()
+            .ConnectionString("host=rabbitmq")
+            .UseConventionalRoutingTopology()
+            .DelayedDelivery().DisableTimeoutManager();
+        #endregion
+        endpointConfiguration.EnableInstallers();
+
+        // The RabbitMQ container starts before endpoints but it may
+        // take several seconds for the broker to become reachable.
+        #region WaitForRabbitBeforeStart
+        await WaitForRabbitToStart()
+            .ConfigureAwait(false);
+
+        var endpointInstance = await Endpoint.Start(endpointConfiguration)
+            .ConfigureAwait(false);
+        #endregion
+
+        Console.WriteLine("Use docker-compose down to stop containers.");
```

Comment:
Is `docker-compose down` the full command that should be used? It's a bit confusing. Would setting it off with backticks like it was actually Markdown help?